src/compiler/config/test/validate-service-worker.spec.ts

```diff
@@ -33,161 +30,108 @@ describe('validateServiceWorker', () => {
    * all over if we try to just access it directly. So instead, do a little
    * check to see if it's falsy. If not, we return it, and if it is we fail the test.
    *
-   * @param target the output target from which we want to pull the serviceWorker
+   * @param serviceWorker The value returned from `validateServiceWorker`
    * @returns a serviceWorker object or `void`, with a `void` return being
    * accompanied by a manually-triggered test failure.
    */
-  function getServiceWorker(target: OutputTargetWww) {
-    if (target.serviceWorker) {
-      return target.serviceWorker;
+  function getServiceWorker(serviceWorker: d.ValidatedOutputTargetWww['serviceWorker']) {
```

Comment:
this function name now seems a bit out-of-date, maybe `ensureServiceWorkerIsDefined` ? or something along those lines

also do we have a generic null-guard function like this hanging around somewhere? I thought maybe we did...

Reply:
Good call on the function, the comment was outdated too so I'll get that fixed up. I'm not sure about the null guard. Haven't seen anything like that, but I can swap it out if you know where that might be